modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRestBuilderTest.java

```diff
@@ -0,0 +1,191 @@
+		}
+
+		Assert.assertTrue(gradleDistribution.contains(GRADLE_WRAPPER_VERSION));
+
+		_gradleDistribution = URI.create(gradleDistribution);
+	}
+
+	@Test
+	public void testBuildTemplateRestBuilderCheckExports() throws Exception {
+		String liferayVersion = getDefaultLiferayVersion();
+		String name = "guestbook";
+		String packageName = "com.liferay.docs.guestbook";
+		String template = "rest-builder";
+
+		File gradleWorkspaceDir = buildWorkspace(
+			temporaryFolder, "gradle", "gradleWS", liferayVersion,
+			mavenExecutor);
+
+		File gradleWorkspaceModulesDir = new File(
+			gradleWorkspaceDir, "modules");
+
+		File gradleProjectDir = buildTemplateWithGradle(
+			gradleWorkspaceModulesDir, template, name, "--package-name",
+			packageName, "--liferay-version", liferayVersion);
+
+		File gradleConfigYAML = new File(
+			new File(gradleProjectDir, name + "-impl"), "rest-config.yaml");
+
+		Assert.assertTrue(gradleConfigYAML.exists());
+
+		File gradleOpenAPIYAML = new File(
+			new File(gradleProjectDir, name + "-impl"), "rest-openapi.yaml");
+
+		Assert.assertTrue(gradleOpenAPIYAML.exists());
+
+		File mavenWorkspaceDir = buildWorkspace(
+			temporaryFolder, "maven", "mavenWS", liferayVersion, mavenExecutor);
+
+		File mavenModulesDir = new File(mavenWorkspaceDir, "modules");
+
+		File mavenProjectDir = buildTemplateWithMaven(
+			mavenModulesDir, mavenModulesDir, template, name, "com.test",
+			mavenExecutor, "-Dpackage=" + packageName,
+			"-DliferayVersion=" + liferayVersion);
+
+		File mavenConfigYAML = new File(
+			new File(mavenProjectDir, name + "-impl"), "rest-config.yaml");
+
+		Assert.assertTrue(mavenConfigYAML.exists());
+
+		File mavenOpenAPIYAML = new File(
+			new File(mavenProjectDir, name + "-impl"), "rest-openapi.yaml");
+
+		Assert.assertTrue(mavenOpenAPIYAML.exists());
+
+		Assert.assertArrayEquals(
+			Files.readAllBytes(gradleConfigYAML.toPath()),
+			Files.readAllBytes(mavenConfigYAML.toPath()));
+		Assert.assertArrayEquals(
+			Files.readAllBytes(gradleOpenAPIYAML.toPath()),
+			Files.readAllBytes(mavenOpenAPIYAML.toPath()));
+
+		testContains(
+			gradleProjectDir, name + "-api/bnd.bnd", "Export-Package:\\",
+			packageName + ".dto.v1_0,\\", packageName + ".resource.v1_0");
+	}
+
+	@Test
+	public void testCompareRestBuilderPluginVersions() throws Exception {
+		String liferayVersion = getDefaultLiferayVersion();
+		String name = "sample";
+		String packageName = "com.test.sample";
+		String implProjectName = name + "-impl";
+		String template = "rest-builder";
+
+		File gradleWorkspaceDir = buildWorkspace(
+			temporaryFolder, "gradle", "gradleWS", liferayVersion,
+			mavenExecutor);
+
+		File gradleWorkspaceModulesDir = new File(
+			gradleWorkspaceDir, "modules");
+
+		buildTemplateWithGradle(
+			gradleWorkspaceModulesDir, template, name, "--package-name",
+			packageName, "--liferay-version", liferayVersion);
+
+		Optional<String> gradleResult = executeGradle(
```

Comment:
because this test invokes gradle, its best to use `isBuildProjects`

Reply:
I understand. Change it to avoid gradle execution if isBuildProjects is false